src/App/Validator/Role/Update.php

```diff
@@ -36,6 +38,15 @@ protected function getRules()
 		];
 	}
 
+	public function checkRolesEnabled(\Kohana\Validation\Validation $validation, $permissions)
```

Comment:
Couple of things:
1. Don't you need to add this into `getRules` above?
2. Should updates fail entirely? or should we be able to rename roles in the free tier?

Reply:
Just realised this is going back to the lumen branch. You need to change the interface to rely on `Ushahidi\Core\Tool\ValidationEngine` instead. It should work as written otherwise

Reply:
> Should updates fail entirely? or should we be able to rename roles in the free tier?
nope no changes to roles at all allowed

>Don't you need to add this into getRules above?
yep

Reply:
> You need to change the interface to rely on Ushahidi\Core\Tool\ValidationEngine instead.
Can you link an example from the other validators, not sure I follow.